llama-index-integrations/embeddings/llama-index-embeddings-bedrock/llama_index/embeddings/bedrock/base.py

```diff
@@ -402,7 +401,6 @@ def _get_request_body(
                 {
                     "texts": payload,
                     "input_type": input_types[input_type],
-                    "truncate": "NONE",
```

Comment:
interesting!

have you tested if any of the other values work? "START", or "END"?

Reply:
@nerdai: The other values are also not working as expected. If you send a text with more than 2048 characters, it is not truncated, but you will receive the error:

_

> A client error occured: Malformed input request: #/texts/1: expected maxLength: 2048, actual: 4920, please reformat your input and try again.

_

If you are interested in testing it yourself, you can run the attached Python script:
[cohere_test.zip](https://github.com/run-llama/llama_index/files/15287327/cohere_test.zip) - Replace your AWS credentials and region in lines 33, 34, and 35.

f you use "truncate": "NONE" in line 70, you will get the "Input too long" error. If you remove the parameters, it works well.

If you send a very long text in text 2 (more than 2048 characters), "truncate": "START" or "END" doesn't work as expected.